### ISSUE TITLE
Typo on manage domains

### ISSUE THREAD

**Issue by ScottPeterJohnson** [ScottPeterJohnson/purelymail-issues#18]
(This issue was imported from Gitea)
**jeremy** on April 3, 2019:
https://purelymail.com/manage/domains

> And now, for the records:

> This is a MX record, with a value (host name) of  mailserver.purelymail.com The trailing period is required. Any value for "Priority" fields will do.

You deleted the trailing period in mailserver.purelymail.com. when adding code formatting.

Actually all the DMARC and DKIM have the same problem. the `host` entries should not have trailing periods (some do) and the `value` entries should all have trailing periods (most don't).

**Comment by ScottPeterJohnson** [comment]
Comment by **jeremy** on April 3, 2019:
Also the "Add new" section can go off the page when trying to add a long entry (like the ownership proof)

![image](/attachments/e174b9ca-44a5-4dca-ac92-2e6f341d7f6d)

**Comment by ScottPeterJohnson** [comment]
Comment by **jeremy** on April 3, 2019:
Also also, your manage domains page is hard to read in general for newbies. Buying this domain was the first exposure I've had to DNS and I've spent at least 10 hours learning the basics, and your page was still a bit confusing.

I wrote up the complete how-to for Google domains, please add it? (where obviously the ownership proof needs to refresh and the things in ` should be code formatted.)


Go to your google domains DNS page:  
https://domains.google.com/m/registrar/\<yourDomain\>/dns  
Scroll to the bottom of the page, and enter custom resource records:  
1. Choose the `MX` record type from the dropdown and enter `10 mailserver.purelymail.com.` in the "Mail server" field. Click `Add`.
2. Choose the `TXT` record type from the dropdown and enter `v=spf1 include:_spf.purelymail.com ~all` in the "Text" field. Click the `+` icon next to the "Text" field. Enter `purelymail_ownership_proof=f0ddd9ba6af62fe0e1ef021dd5cb3cb753be06b8cb96f21e7693dbbd0f605c9e77bd8ab74298f66a10062e59d80f7d97ec0781113388d23cdf1c1a8f5e95e2c61554342202383`. Click `Add`.

3. DKIM and DMARC. Make 4 `CNAME` records by selecting `CNAME` from the dropdown record type, putting the `host` in the "@" field and the `value` in the "Domain name" field. The trailing periods are, as always, required. Click the `Add` button after every record  
  a. `host`: `purelymail1._domainkey`; `value`: `key1._dkimroot.purelymail.com.`  
  b. `host`: `purelymail2._domainkey`; `value`: `key2._dkimroot.purelymail.com.`  
  c. `host`: `purelymail3._domainkey`; `value`: `key3._dkimroot.purelymail.com.`  
  d. `host`: `_dmarc`; `value`: `_dmarcroot.purelymail.com.`

**Comment by ScottPeterJohnson** [comment]
Comment by **Scott** on April 3, 2019:
Thank you so much for your feedback! I really appreciate that you took the time to write instructions for the Google DNS. I'll add them soon.

> Actually all the DMARC and DKIM have the same problem. the host entries should not have trailing periods (some do) and the value entries should all have trailing periods (most don’t).

I'll admit I was slightly confused as to when the trailing . is required. You appear to be right that it should be all CNAME and MX records (but not the SPF value): https://stackexchange.github.io/dnscontrol/why-the-dot

> Also the “Add new” section can go off the page when trying to add a long entry (like the ownership proof)

Thanks! I'll force wrapping there.

> Also also, your manage domains page is hard to read in general for newbies. Buying this domain was the first exposure I’ve had to DNS and I’ve spent at least 10 hours learning the basics, and your page was still a bit confusing.

Sorry that it took so long for you to figure out DNS. I'm not terribly happy with that page either. 

Part of the problem is, as you might have guessed, that it's a lot of meaningless complexity to dump on someone who's never even heard of DNS before. Much of it is inevitable, but adding eight records still sucks. It's even more annoying that the SPF and MX records can't be CNAMEs like everything else, and they contain spaces, which are a giant can of worms.

Another part is that all 4 of the DNS registrars I've looked at have been slightly different. [You can see how Google ended up handling this.](https://support.google.com/a/topic/1409901) I really wish the DNS industry would come up with a standard for authorizing addition of DNS records.

**Comment by ScottPeterJohnson** [comment]
Comment by **Scott** on April 3, 2019:
> Lastly, can you please change the issue tracker so that hitting space twice doesn’t automatically insert a period? double space is useful for linebreaks in markdown, and it was hard to format this when it kept adding unnecessary periods.

I have no idea about this, and it doesn't work that way at least for me. Are you using a Chromebook? I think those do this for you.

**Comment by ScottPeterJohnson** [comment]
Comment by **jeremy** on April 3, 2019:
Oops. You're right, it was a mac keyboard setting. Ignore that bit.

**Comment by ScottPeterJohnson** [comment]
Comment by **Scott** on April 15, 2019:
I fixed the trailing period typoes, added warnings if a record is missing them (previously it accepted with or without; without might cause issues. All of the registrars I have add it automatically though), fixed the other issues you mentioned, and added the guide you wrote (thanks!). 
I also added a little bit more guidance, with a short explanation of what DNS is. I think people will still be overwhelmed, but it's better maybe. Hopefully they'll ask for support, so I can refine the page based off their questions.

Take a look if you have time! I'd appreciate any feedback.
